content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,42 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+And update the `process.py` file to count the number of digits.
+
+```python
+import sys
+num_digits = 0
+with open(sys.argv[1], 'r') as f:
+    for number in f:
+        num_digits += len(number) - 1
+print("Number of digits:",end=" ")
+print(num_digits)
+```
```

Comment:
This change is way to complex, why not something simple again like in lines 227-34? Also the Python code isn't properly formatted...

Reply:
I'm not even sure this update is needed...

Reply:
I guess it does clarify the example. Let's keep it! But please simplify the Python change. Should be 1 line.

Reply:
Continues in https://github.com/iterative/dvc.org/pull/1624#pullrequestreview-457247058 ...